assets/css/editor.scss

```diff
@@ -1,6 +1,3 @@
-// Import the woocommerce components stylesheet
-@import "~@woocommerce/components/build-style/style.css";
-
```

Comment:
I'm seeing some styling issues in that component.

![image](https://user-images.githubusercontent.com/6165348/90760032-2da83380-e2d9-11ea-9c41-ea27ea517156.png)
![image](https://user-images.githubusercontent.com/6165348/90760041-300a8d80-e2d9-11ea-91ff-ea6d65f629e6.png)

I think, due to our special styling behavior, we might want to leave this file or move it somewhere in our webpack build cc @Aljullu 

Also, from my point of view, if we're only using this component alone, I'd say leave the package as is (load from node_module) and work on tree shaking the library, we're now loading the whole package, and we only need a fraction of it.

on main
![image](https://user-images.githubusercontent.com/6165348/90760358-a7402180-e2d9-11ea-950e-0252719aafa4.png)

Reply:
> Also, from my point of view, if we're only using this component alone, I'd say leave the package as is (load from node_module) and work on tree shaking the library, we're now loading the whole package, and we only need a fraction of it.

I agree.

Reply:
thinks work fine when I leave the stylesheet in place
![image](https://user-images.githubusercontent.com/6165348/90761763-cd66c100-e2db-11ea-84c9-129af48214be.png)